Makefile

```diff
@@ -101,6 +107,18 @@ fix: build
 lint: deps
 	$(GOPATH1)/bin/golangci-lint run -c .golangci.yml
 
+# NOTE: tidy will download and install go version GOLANG_VERSION_BUILD 
+# 		if this isn't the system's current version
```

Comment:
It's nice that `go install` can manage this, but it still seems a little strange to me that this is done in the Makefile.

I'm not completely oppose to the change here, but exiting with an error that tells you the go version is wrong would be my preference. Maybe even split it out with a `check` target which checks your environment, and add that as a dependency to other things which might be wrongish with the wrong version of go installed.

Reply:
c7e31fe657df6d31e708b228b4f4f6ac6246b457